packages/discord.js/src/structures/ApplicationCommand.js

```diff
@@ -123,7 +123,7 @@ class ApplicationCommand extends Base {
        * The options of this command
        * @type {ApplicationCommandOption[]}
        */
-      this.options = data.options.map(o => this.constructor.transformOption(o, true));
+      this.options = data.options.map(opt => this.constructor.transformOption(opt, true));
```

Comment:
Why not expand this to option too like in other files?

Reply:
Because option is already declared in the upper scope

Reply:
Where?

Reply:
I remember it throwing an error, but now that I tried again, it seems to have no issues, weird.